app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        try:
+            if not visitor["vaccine"]:
+                raise NotVaccinatedError("NotVaccinatedError")

```

Comment:
The error message 'NotVaccinatedError' is not descriptive. According to the checklist, you should provide a more descriptive message when raising exceptions. Consider including more context about the error.